lib/HTML/FormBuilder/Base.pm

```diff
@@ -73,7 +73,7 @@ sub _build_element_and_attributes {
 
         # skip attributes that are not intended for HTML
         next if ($key =~ /^(?:option|text|hide_required_text|localize|wrap_in_div_class)/i);
-        if ($attributes->{$key}) {
+        if (defined $attributes->{$key} and ('' . $attributes->{$key}) ne '') {
```

Comment:
Since there's already a `next` prior to this, why not just test it from there?

```perl
next unless ($key !~ /.../i) and $attributes->{$key} ne '';
```

Reply:
I think `next` is better, I prefer to set one for each condition for readability.